pkg/storaged/mdraid/create-dialog.jsx

```diff
@@ -110,12 +110,25 @@ export function create_mdraid() {
         ],
         Action: {
             Title: _("Create"),
-            action: function (vals) {
-                return prepare_available_spaces(client, vals.disks).then(paths => {
-                    return client.manager.MDRaidCreate(paths, vals.level,
-                                                       vals.name, (vals.chunk || 0) * 1024,
-                                                       { });
-                });
+            action: async function (vals) {
+                if (client.in_anaconda_mode()) {
+                    // HACK - Anaconda has trouble detecting invalid
```

Comment:
The problem is not that anaconda has problem detecting invalid configuration, the problem is that cockpit MDRAID devices are always using metadata 1.2, and bootloader partitions are on such MDRAID devices the system will end up being unbootable.

Reply:
You wrote in https://bugzilla.redhat.com/show_bug.cgi?id=2353195:

> Therefore Software RAID on top of disks was never tested or supported in anaconda. 

This PR makes it so that Anaconda is not exposed to Software RAID on top of disks.  If Anaconda has no problem with Software RAID on top of disks, then we don't need this PR.